Makefile

```diff
@@ -7,7 +7,7 @@ help:
 # -- variables --------------------------------------------------------------------------------------
 
 WARNINGS=RUSTDOCFLAGS="-D warnings"
-ALL_FEATURES_BUT_ASYNC=--features concurrent,testing
+ALL_FEATURES=--features concurrent,testing
```

Comment:
`make doc` still uses `ALL_FEATURES_BUT_ASYNC` - I think that needs to be updated?

Also, should we remove this and inline `--all-features` in the commands that use it instead? That would also explicitly activate `std`, but that seems fine. The commands that use `ALL_FEATURES` imply `std` while we have specific `no-std` commands that don't use `ALL_FEATURES`.